### ISSUE TITLE
🐛 CSS parser fails to parse * in variable

### ISSUE THREAD

**Issue by condemil** [biomejs/biome#4794]
### Environment information

```block
vscode embedded biome v1.9.4
```

### What happened?

The following tailwind v4 css syntax breaks the parser:

```css
--color-*: initial;
```

### Expected result

Parser doesn't brake and accepts asterisk in the variable

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by condemil** [comment]
And similar issue with:

```css
@import "tailwindcss" source(none);
```

**Comment by ematipico** [comment]
Is it valid, stable, CSS syntax?

**Comment by condemil** [comment]
I did a quick check and it doesn't look like it's a part of CSS spec.

**Comment by dugajean** [comment]
I'm getting it for `.`. We should perhaps have the option to disable this rule for certain lines.

<img width="691" alt="Screenshot 2024-12-27 at 12 02 04 PM" src="https://github.com/user-attachments/assets/087149e3-96a0-4629-bc4b-2a3e73ba38be" />

**Comment by ematipico** [comment]
This isn't a rule, you can see that from the tooltip you just posted, it's written `biome(parse)`.

Biome couldn't parse the file because tailwind uses syntax that isn't still stable.

Best thing is to ignore the file.

**Comment by malobre** [comment]
You need to escape the asterisk, i.e: `--color-\*: initial;`. Tailwind *should* understand this syntax, otherwise it's a bug on their end.

**Comment by malobre** [comment]
@condemil Latest tailwind beta (`4.0.0.beta-10`) has added support for the escaped asterisk discussed here.
